numba/errors.py

```diff
@@ -110,11 +335,12 @@ def contexts(self):
 
     def add_context(self, msg):
         """
-        Add contextual info.  The execption message is expanded with the new
-        contextual info.
+        Add contextual info.  The exeception message is expanded with the new
```

Comment:
"exception"

Reply:
Seems like I managed to correct that to another incorrect spelling :) thanks, fixed.